src/lib/Libdis/ps_dis.c

```diff
@@ -264,25 +264,25 @@ vn_decode_DIS_V3(int fd, int *rcp)
 
 	if ((vnlp = malloc(sizeof(vnl_t))) == NULL) {
```

Comment:
Should use calloc to initialize vnlp in case free_and_return() gets called. Or memset it to zeros.